user-interface/src/main/resources/application.properties

```diff
@@ -131,5 +132,11 @@ qbic.communication.documentation.url=${DOCUMENTATION_URL:https://qbicsoftware.gi
 qbic.communication.contact.email=${CONTACT_EMAIL:support@qbic.zendesk.com}
 qbic.communication.contact.subject=${CONTACT_SUBJECT:Data Management Question}
 qbic.communication.api.url=${API_URL:https://download.qbic.uni-tuebingen.de/swagger-ui.html}
-################# Session handling ############################################
+################# Session servlet handling ############################################
 server.servlet.session.timeout=${SESSION_TIMEOUT:30m}
+vaadin.closeIdleSessions=${SESSION_CLOSEIDLESESSIONS:true}
+vaadin.heartbeatInterval=${SESSION_HEARTBEATINTERVAL:300}
```

Comment:
seconds, milliseconds, nanoseconds? consider adding a comment

Reply:
Sadly we can't specify the time unit as a suffix but this is described already in the additional information contained in 
`user-interface/src/main/resources/META-INF/additional-spring-configuration-metadata.json`